crates/bitwarden/src/client/client.rs

```diff
@@ -79,6 +80,30 @@ pub struct Client {
     encryption_settings: Option<EncryptionSettings>,
 }
 
+#[cfg(feature = "secrets")]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ClientState {
+    pub token: String,
+    pub token_expiry_timestamp: i64,
```

Comment:
This is part of the jwt, so we should be able to remove it from the persisted state.

Reply:
Resolved in [566545d](https://github.com/bitwarden/sdk/pull/388/commits/566545d475abb1fd04eb825090e4227df841f4ed)